Board.cs

```diff
@@ -1,18 +1,64 @@
+using System.Collections.Generic;
+using System.Linq;
+
 namespace TicTacToe
 {
     public class Board
     {
         private static IConsoleWriter _writer;
-        static char[] boardArr = { '1', '2', '3', '4', '5', '6', '7', '8', '9' };
+        private static List<object> _boardList;
 
         public Board(IConsoleWriter IWriter)
         {
+            _boardList = new List<object>(){ 1, 2, 3, 4, 5, 6, 7, 8, 9 };
             _writer = IWriter;
         }
         
         public void DisplayBoard()
         {
-            _writer.PrintBoard(boardArr);
+            _writer.PrintBoard(_boardList);
+        }
+
+        public void MakeMove(int cell, char playerSymbol)
+        {
+            _boardList[cell - 1] = playerSymbol;
+        }
+
+        public bool IsValidMove(int cell)
+        {
+            if (Enumerable.Range(1, 9).Contains(cell))
+            {
+                var validMoves = AvailableMoves();
+                var playerMove = _boardList[cell - 1];
+                return validMoves.Contains(playerMove);
+            }
+
+            return false;
+        }
+
+        public List<object> Spaces()
+        {
+            return _boardList;
+        }
+        
+        public List<object> AvailableMoves()
+        {
+            var availableList = new List<object>();
+
+            foreach (object cell in _boardList)
+            {
+                if (cell is int)
+                {
+                    availableList.Add(cell);
+                }
+            }
+
+            return availableList;
+        }
+
+        public int TurnCount()
+        {
+            return 9 - AvailableMoves().Count;

```

Comment:
Another magic number here. Think about how you could pull this out to make this class configurable, or at the least make this a constant with a name that tells us more about it.